titiler/settings.py

```diff
@@ -0,0 +1,10 @@
+"""Application environment variables"""
+
+from starlette.config import Config
+
+config = Config()
```

Comment:
I know this is not part of the scope of this PR but it will be great to unify `settings` and `titiler.core.config`

Reply:
Agreed, I think I will move the contents of `settings` into `titiler.core.config`.

Reply:
or keep `titiler.settings` only? up to you ;-)

Reply:
I think `settings` should be used in the future especially for environment vars but I think this PR to stick to mosaics and follow the current repo structure (ex. I've been trying to avoid deduping code in other routes even though it is very tempting)

Reply:
https://github.com/developmentseed/titiler/pull/40/commits/60ede4cb7bafa1b6b4e748dc784cb987493296cd